### ISSUE TITLE
Quickly pressing buttons in shooting mode causes functions to get stuck on

### ISSUE THREAD

**Issue by matthudsonau** [gkoh/furble#63]
Equipment: M5StickC PLUS, GFX 100ii

When in shooting mode, quickly tapping a button only seems to trigger the shutterPress/focusPress functions, and the shutter/focus is locked on until the button is long pressed (another quick press keeps the function locked on)

I actually find this behaviour to be very useful, however trying to 'unstick' the buttons is annoying. It would be a lot better if a second quick press simply turned the function off

**Comment by gkoh** [comment]
@matthudsonau Thank you for the bug report!
That was embarrassingly easy to reproduce :neutral_face: 
I have a fix and can release a v2-alpha image if you want to test.

Further, I'm interested in this comment _"I actually find this behaviour to be very useful"_.
Could you elaborate?

The current button handling is designed for CL/CH shooting, where you hold for continuous snapping.

**Comment by matthudsonau** [comment]
On my previous remote (wired canon thing on my 50S), I would lock the shutter so it would shoot continuously. Very useful when shooting lightning or astro. Can't do that on the 100 ii (new remote connector)

I did shoot some lightning around a month ago, and having to hold the button in to keep shooting got very annoying within a couple of minutes. Similar with astro (although in that case I used the internal intervalometer, but it's a hop through the menus to set up and configure every time I want to move the camera)
